### DISCUSSION TITLE
how to run yolo detection code on gpu?

### DISCUSSION THREAD

**Discussion by StefanoGarzoni** [roboflow/notebooks#121] [Q&A]
I ran the code of this video : https://www.youtube.com/watch?v=Mi9iHFd0_Bo
with the addition of using a yolo custom object detection algorithm (yolov8m trained on 300 images)
the code works and detects the object, but as soon as I move an object faster, it disappears from tracking and rightly the linecounter doesn't work...
I assume that this is perhaps due to the fact that I should make it run on the gpu, but I don't know how to do it...
If this shouldn't be the problem or if it should be this, could someone explain to me what I should do to solve this problem?
Thank you so much as always to anyone who replies!
You are fantastic, I owe you a lot for my personal growth on the subject!
look forward to your help.

**Comment by SkalskiP** [comment]
Hi, @Stefano0513 👋🏻 could you share a sample of the video result?

**Reply by StefanoGarzoni** [reply]
https://github.com/roboflow/notebooks/assets/131482737/ca8e4a6e-9437-4257-8804-2057c406c15d

here is the video test, as you can see as soon as I increase the speed (but even less than that) it goes jerky and with slower object scrolling speeds you can also see how it loses tracking and consequently the linecounter does not work.
help me please, Thank you so much!

**Reply by StefanoGarzoni** [reply]
hello everyone, in these days I managed to solve it by changing the webcam by buying a new one, now the output video shown is fluid, but in any case when there is an important speed, the tracker loses the object and this causes the linecounter doesn't count it... maybe i should retrain model better (yolov8m trained on 300 img) or should i change base model?
